packages/channel-provider/src/channel-provider.ts

```diff
@@ -1,15 +1,104 @@
-import EventEmitter, {ListenerFn} from 'eventemitter3';
+import debug from 'debug';
+import EventEmitter from 'eventemitter3';
+import {JsonRPCRequest} from 'web3/providers';
```

Comment:
Is it possible to a void using `web3`?

Reply:
I'm just borrowing type definitions, but sure. I could declare them on my own or use something else.

Reply:
Done via 7376f32.